charts/ai-software-templates/chatbot/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 kubeVersion: ">= 1.27.0-0"
 annotations:
-  charts.openshift.io/name: Chatbot AI Software Template
-description: A helm chart for the AI Software Templates. For more information please check https://github.com/redhat-ai-dev/ai-lab-helm-charts.git
-name: chatbot-ai-software-template
+  charts.openshift.io/name: Chatbot AI Sample
+description: A helm chart for the Chatbot AI Sample app. For more information please check https://github.com/redhat-ai-dev/ai-lab-helm-charts.git
+name: chatbot-ai-sample
```

Comment:
If someone runs a `helm upgrade` with an existing installation and the chart name has changed, what effect will that have?

Reply:
The helm chart installation relies on the `Release.Name` value. We don't use the chart name any where so I woulnd't expect an impact there.

Reply:
Works for me, thanks.

Reply:
Thanks for the help!